cmake/profiles/debug.cmake

```diff
@@ -51,6 +54,7 @@ if(MBED_TOOLCHAIN STREQUAL "GCC_ARM")
 elseif(MBED_TOOLCHAIN STREQUAL "ARM")
     list(APPEND c_compile_options
         "-std=gnu11"
+        "-O1"
```

Comment:
Where is this `-O1` coming from in the old tools? I'd have expected `-O0` for the debug profile.

Reply:
yes correct, this is matching what we use

Reply:
OK, I see this is in tools/profiles/debug.json.